### ISSUE TITLE
Problem installation

### ISSUE THREAD

**Issue by victorballester7** [PlisJan/bingwallpaperinfo#3]
Hi,

I very like your idea, but I'm having troubles installing the widget. I copied the folder `package` into `.local/share/plasma/plasmoids/`, and then I can see the widgets on the window `Add Widgets`, but then when I drag it and drop the widget to the desktop, it just disappears.

How I can properly install it?

Thanks,

**Comment by PlisJan** [comment]
Hi there,
sorry for my extremely late answer. I just haven't found time to investigate and fix your issue until now :)
I have ported the widget to KDE6 and added build instructions in the README.md file.
But all in all your workflow should also work. I just had some porting issues with the new KDE versions.

Could you tell me if it works by now?

**Comment by victorballester7** [comment]
Sorry, I ended up creating my own Kde widget (which I don't use anymore though...).

**Comment by PlisJan** [comment]
Ah, great, have you published yours on Github?

**Comment by victorballester7** [comment]
Yes, there you have: [https://github.com/victorballester7/title-bing-wallpaper](https://github.com/victorballester7/title-bing-wallpaper)

Pd: Sorry for my bad and simple writing, I don't have my laptop available right now and with the mobile app is terrible. :(
